Configuration/ConsulRx.Configuration.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <Version>1.1.0</Version>
+    <Version>1.2.0</Version>
```

Comment:
There are no changes to this project, but some consumers reference `ConsulRx.Configuration` (as opposed to `ConsulRx` and we want them to see an available update as well)